discovery-provider/src/queries/get_users_cnode.py

```diff
@@ -36,13 +36,18 @@ def get_users_cnode(cnode_endpoint_string, replica_type=ReplicaType.PRIMARY):
                 "wallet",
```

Comment:
this helper function is consumed in two diff routes, just make sure both still work

Reply:
hmm I think that since this route is purely additive to the spIds, we shouldn't be experiencing any breakage